pkg/service/aws/volume.go

```diff
@@ -171,7 +182,11 @@ func (svc awsController) CreateSnapshot(ctx context.Context, req *proto.CreateSn
 	}
 
 	tags := awsTags(labels)
-
+	name := fmt.Sprintf("snapof-%s", volumeid)
```

Comment:
are we using the name format somewhere to identify something in any place? if yes, can this break backwards compatibility?

Reply:
no, just for reference and ease of spotting in web console. No use.